### DISCUSSION TITLE
Why so complicated? Help.

### DISCUSSION THREAD

**Discussion by stefanw1337** [linux-surface/linux-surface#1459] [Q&A] [answered]
So you'd think this would be easy. 
You read, download arch live iso. But then there's no actual iso file.
You read, click here for guides, then there's no windows guides, you all know Surface IS windows by default, right?

Why make it so hard? So I downloaded Rufus, which I have used before, renamed the arch live pkg.tar.zst file to iso. And no, it's still not that easy. So help? How to easily do this. I have used Gentoo guide before, but that was more detailed.

**Comment by dev-nis** [comment, marked as answer]
I think you misunderstood something. You can't download.iso-files of this project, because it is a kernel, not an OS. So, if you want to install Arch, you need to download it from their site, flash it to a USB, e.g. with Rufus,boot and install it and then follow the instructions here to install the Surface-Kernel on Arch

**Comment by stefanw1337** [comment]
Well, having issues booting up arch live Bootable USB drive made with both Rufus and uNetbootin. I've checked the bios and tried from windows to make the surface boot the USB. And I can see the lights blink on the stick. But it simply doesn't want to boot up anything other than windows. So any guides I've seen doesn't cover this issue. I tried with an SD card as well, still doesn't want to boot.
